src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        ColorSupplier supplier = new ColorSupplier();
+        for (int i = 0; i < 3; i++) {
+            System.out.println(supplier.getRandomColor());
+            System.out.println(lottery.getRandomBall());
+        }
         // create three balls using class Lottery and print information about them in console
     }
```

Comment:
There seems to be an unnecessary empty line at the end of the main method. It's a good practice to remove any redundant empty lines to keep the code clean.